woocommerce/woocommerce
# Hide LYS launch newsletter panel when feature is disabled

### Changes proposed in this Pull Request:

Slack: p1734418349811269-slack-C01SFMVEYAK

Follow-up to https://github.com/woocommerce/woocommerce/pull/52650

Newsletter panel is not meant to be shown when `coming-soon-newsletter-template` feature flag is disabled.

This PR ensures it is only shown when the feature flag is enabled.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In a fresh site, go to `Appearance > Editor > Templates`
2. Search for `Page: Coming soon` and click on it
3. Observe no `Launch Newsletter` panel on the right sidebar ([screenshot](https://github.com/user-attachments/assets/c158b2c5-5d41-42c4-bd33-825d5c7e7685))
4. In a new tab, go to `Tools > WCA Test Helper > Features`
5. Enable `coming-soon-newsletter-template` feature flag
6. Go back to the previous tab and refresh
7. Observe `Launch Newsletter` panel is shown on the right sidebar ([screenshot](https://github.com/user-attachments/assets/6a71c4b9-b533-449d-8276-47e8e7dd0b0b))



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
